src/vma/sock/sockinfo.h

```diff
@@ -123,11 +123,13 @@ class sockinfo : public socket_fd_api, public pkt_rcvr_sink, public pkt_sndr_sou
 
 	inline bool tcp_flow_is_5t(void) { return m_tcp_flow_is_5t; }
 	inline void set_tcp_flow_is_5t(void) { m_tcp_flow_is_5t = true; }
-	inline void set_flow_tag(int flow_tag_id) {
-		if ( flow_tag_id && (flow_tag_id != FLOW_TAG_MASK)) {
+	inline bool set_flow_tag(uint32_t flow_tag_id) {
+		if (flow_tag_id && (flow_tag_id != FLOW_TAG_MASK)) {
 			m_flow_tag_id = flow_tag_id;
 			m_flow_tag_enabled = true;
+			return true;
 		}
+		return false;
```

Comment:
I think that this code does not take into account that flow_tag_id should be limited by 20 bit (FLOW_TAG_MASK).
flow_tag_id can not exceed FLOW_TAG_MASK

Reply:
done